.github/workflows/tests.yml

```diff
@@ -14,9 +14,9 @@ jobs:
     strategy:
       matrix:
         go:
-          - ^1.13
```

Comment:
As mentioned, we may leave it until the compatibility would be dropped by some new language feature

Reply:
To be honest I don't think we need to support even 1.15 (or well, maybe after 1.16.1 release it should be ok to drop 1.15) as usually people migrate to latest golang rather quickly, but I would leave 1.14 as "until something breaks" and would try to maintain compatibility with 1.15 while it's still officially supported.